### PR TITLE
Add versions of '#core' and '#nocore' that dont use the c-word so we can surface the options to the public.

### PR DESCRIPTION
**Pull request by michaelwmcnamara**
At the moment, users who have opted out of the enhanced pages are unable to comment without opting back in to all pages.
We have the ability to display a particular page as fully enhanced even though the default setting for our browser is set to show the non-enhanced (core pages). This is currently done by appending '#nocore' to the url.

Users running the core opt in have asked if comments will be added to the non-enhanced pages. While we wrestle with this, I would like to add information about the '#nocore' option to our user help page - https://www.theguardian.com/info/2015/sep/22/making-theguardiancom-work-best-for-you
- so that if a user feels particularly moved to comment on a page, or if they would like to see an interactive running properly, they have an easy way to do so. 

The problem at the moment is the naming of this option still uses the old 'core' / 'nocore' terminology, which we have been asked to steer clear of.

This change adds two extra # options '#featuresoff' and '#featureson' which do the same job as '#core' and '#nocore' but will make more sense to users (and avoid using the dreaded "core").

### CODE REVIEW

**Comment by michaelwmcnamara** [comment]
Other option I was toying with was to replace '#core' and '#nocore' entirely with '#featuresoff' and '#featureson'. Let me know if you think this is a better approach.

File: common/app/templates/headerInlineJS/shouldEnhance.scala.js [hunk 1]
```diff
@@ -4,8 +4,8 @@
 (function (navigator, window) {
     // Enable manual optin to core functionality/optout of enhancement
     var personPrefersCore = function () {
-        if (window.location.hash === '#core' || window.location.hash === '#gu.prefs.force-core=on') return true;
-        if (window.location.hash === '#nocore' || window.location.hash === '#gu.prefs.force-core=off') return false;
+        if (window.location.hash === '#featuresoff' || window.location.hash === '#core' || window.location.hash === '#gu.prefs.force-core=on') return true;
```
Comment (sndrs) [inline common/app/templates/headerInlineJS/shouldEnhance.scala.js]: would be better to cache calls to `window.location.hash` as this is critical and inlined?

File: common/app/templates/headerInlineJS/shouldEnhance.scala.js [hunk 2]
```diff
@@ -4,8 +4,9 @@
 (function (navigator, window) {
     // Enable manual optin to core functionality/optout of enhancement
     var personPrefersCore = function () {
-        if (window.location.hash === '#core' || window.location.hash === '#gu.prefs.force-core=on') return true;
-        if (window.location.hash === '#nocore' || window.location.hash === '#gu.prefs.force-core=off') return false;
+        var locationHash = window.location.hash
```
Comment (sndrs) [inline common/app/templates/headerInlineJS/shouldEnhance.scala.js]: needs a `;` at the end of the line

**Comment by sndrs** [comment]
:+1:

**Pull request merged**